go.mod

```diff
@@ -11,15 +11,19 @@ require (
 	github.com/minio/sha256-simd v1.0.1
 	github.com/rs/xid v1.5.0
 	github.com/sirupsen/logrus v1.9.2
+	github.com/stretchr/testify v1.7.0
 	golang.org/x/crypto v0.9.0
```

Comment:
Please don't bring in testify, please use Go's testing package instead.

Reply:
ok

Reply:
Done!